deployments/k8s/helm/signalfx-agent/templates/configmap.yaml

```diff
@@ -109,6 +116,7 @@ data:
       - {{ toYaml . | indent 8 | trim }}
       {{- end }}
       {{- end }}
+{{- end }}
 
     monitors:
 {{- if .Values.configureStandardMonitors }}
```

Comment:
I would prefer if this followed the same pattern as we do for monitors: there would be a config option `configureStandardObservers` that defaults to true that you could turn off  to disable the normal observers, and then provide the `observers` config that would configure those observers, possibly in addition to the standard ones.  Does that make sense?

Reply:
This allows for the possibility that you want the standard observers but also want to add another observer without having to figure out manually what the standard observer config is.

Reply:
I can do that but one of the core issues with leaving the standard observers list is that `discoverAllPods` and `discoverNodes` are hardcoded to `true` if running in serverless mode.

Should I also change that to obey the value of those settings regardless of serverless mode?

Reply:
Yes, in your use case, you should just set `configureStandardObservers: false` and provide your own observer list.